synapse_invite_checker/invite_checker.py

```diff
@@ -801,6 +810,55 @@ async def get_delete_tasks_by_room(self, room_id: str) -> list[ScheduledTask]:
             statuses=statuses,
         )
 
+    async def kick_all_local_users(self, room_id: str) -> None:
+        """
+        First block the room from further interaction from this server, then cause all
+        local members to set their membership to 'leave'
```

Comment:
```suggestion
        First block the room from further interaction from this server, then kick all local users.
```

I don't think you need to explain how kicking works...

Reply:
Fair, patched up